app/src/main/java/com/globant/practice/presentation/presenter/HomePresenter.java

```diff
@@ -0,0 +1,82 @@
+package com.globant.practice.presentation.presenter;
+
+import com.globant.practice.domain.interactor.UserInteractor;
+import com.globant.practice.domain.model.User;
+import com.globant.practice.presentation.view.activity.HomeView;
+import java.util.List;
+import javax.inject.Inject;
+import io.reactivex.Observable;
+import io.reactivex.Observer;
+import io.reactivex.android.schedulers.AndroidSchedulers;
+import io.reactivex.disposables.Disposable;
+import io.reactivex.schedulers.Schedulers;
+
+/**
+ * Contains the necessary methods for the view actions.
+ * Created by jonathan.vargas on 4/04/2017.
+ */
+
+public class HomePresenter {
+
+    private HomeView view;
+    private UserInteractor interactor;
+    private Observable<List<User>> getUsersListObservable;
+
+    /**
+     * Construct method of the HomePresenter, receives a UserInteractor instance for
+     * manage the user data.
+     *
+     * @param interactor
+     */
+    @Inject
+    public HomePresenter(UserInteractor interactor) {
+        this.interactor = interactor;
+    }
+
+    /**
+     * Receives and assign an instance of the view interface.
+     *
+     * @param view view interface reference
+     */
+    public void attachView(HomeView view) {
```

Comment:
These methods **attach** and **detach** are duplicated from previous Presenter. Have you thought of implementing a BasePresenter class with commons methods?